Expose getEAWOfCodePoint (#15)
susisu/meaw: README.md

assert(getEAW("ℵAあＡｱ∀", 2) === "W");
```

### `getEAWOfCodePoint()`

> Similar to `getEAW()`, but takes the code point instead of the string of a character.

Reviewer: ```suggestion
Similar to `getEAW()`, but takes a code point (number) instead of a string.
```

```javascript

> import { getEAWOfCodePoint } from .codePointAt(0)"meaw";

Reviewer: ```suggestion
import { getEAWOfCodePoint } from "meaw";
```

>  *

Reviewer: ```suggestion

```
